src/routes/events/FeaturedEvent.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+    import Button from '$lib/components/Button.svelte';
+
+    export let tags: string[] = [];
+    export let schedule: string[] = [];
```

Comment:
I typically discourage defaulted props in any framework because it's so easy to forget that they affect the component. As a best practice, let's force the caller to always pass in the array.

```suggestion
    // eslint-disable-next-line init-declarations
    export let tags: string[];
    // eslint-disable-next-line init-declarations
    export let schedule: string[];
```

On a separate note, though, perhaps we could also use a `const enum` somewhere so that the `tags` aren't just the generic `string[]`. In terms of type safety, I'm happy to see stricter literals here.

Reply:
I've updated my branch to implement your suggestions. I also added some `const enum`s for Tags, State, and Type in `events.ts` and used them in some components. However, I couldn't really use them in many places, since I couldn't change the `Event` type in the `EventsPanel` due to both the Events and Projects pages depending on it. With that said, I believe it will be necessary for us to rethink the implementation of tag filters when generalizing `EventsPanel` to include the type safety that `enum`s provide to some degree (using unions possibly).

Reply:
Sure! We can do this in a follow-up PR. In the meantime, your implementation works great. 👍